js-jasmine/src/shop.js

```diff
@@ -0,0 +1,31 @@
+const Item = require('./item.js')
+const AgedBrie = require('./agedbrie.js')
+const Ordinary = require('./ordinary_items.js')
+const Conjured = require('./conjured_items.js')
+const Sulfuras = require('./sulfuras.js')
+const Passes = require('./passes.js')
+
+class Classes{
+  'Aged Brie'= AgedBrie
```

Comment:
throwing a syntax error when i try to run the tests:
`SyntaxError: C:/Users/anton/Desktop/gilded_peer/GildedRose-Refactoring-Kata/js-jasmine/src/shop.js: 
Unexpected token (9:13)`